### PR TITLE
Added new module stellar-dispatcher

### PR DESCRIPTION
**Pull request by roeeyud**

### CODE REVIEW

File: packages/stellar-dispatcher/src/index.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import uuid from 'uuid';
+import _ from 'lodash';
```
Comment (sloops77) [inline packages/stellar-dispatcher/src/index.js]: you have to import functions

File: packages/stellar-dispatcher/src/index.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+const stoppersMap = {};
+let stellar;
+
+export function configure(_stellar) {
+    stellar = _stellar; // eslint-disable-line better-mutation/no-mutation
+}
+
+export const dispatch = ({ resource, path, method, channel, reactiveHandler, payload, options }) => {
+    if (method === 'unsubscribe') {
+        const stopper = stoppersMap[payload];
```
Comment (sloops77) [inline packages/stellar-dispatcher/src/index.js]: payload as the key? pls comment what it really is

File: packages/stellar-dispatcher/src/index.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        if (stopper) {
+            stopper();
+        }
+        _.unset(stoppersMap, payload); // eslint-disable-line better-mutation/no-mutating-functions
+        return true;
+    }
+
+    if (method === 'subscribe') {
+        const finalChannel = `op:${_.get(options, 'headers.operationId')}:${channel}`;
```
Comment (sloops77) [inline packages/stellar-dispatcher/src/index.js]: this is a gefen concept. no acceptable in stellar

File: packages/stellar-dispatcher/src/index.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        const response = stellar.request.getReactive(path ? `${resource}:${path}` : resource,
```
Comment (sloops77) [inline packages/stellar-dispatcher/src/index.js]: assign ternary to a const

File: packages/stellar-dispatcher/src/index.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+        return Promise.all([response.onStop, response.results])
+            .then(([stopper, results]) => {
+                const stopperId = uuid();
+                stoppersMap[stopperId] = stopper; // eslint-disable-line better-mutation/no-mutation
+                return ({ stopperId, results });
+            });
+    }
+
+    return stellar.request.request(path ? `${resource}:${path}` : resource, method, payload, options);
```
Comment (sloops77) [inline packages/stellar-dispatcher/src/index.js]: const it

File: packages/stellar-dispatcher/src/index.js [hunk 6]
```diff
@@ -0,0 +1,36 @@
+import uuid from 'uuid';
```
Comment (sloops77) [inline packages/stellar-dispatcher/src/index.js]: you should import a sepcific version of uuid v4 probably

File: packages/stellar-dispatcher/package.json [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+  "scripts": {
+    "ci": "yarn run lint:ci && yarn run test-coverage",
+    "clean": "rimraf lib lib-es6",
+    "compile": "yarn run compile:es6 && yarn run compile:umd",
+    "compile:es6": "babel ./src  --out-dir ./lib-es6",
+    "compile:umd": "webpack -p",
+    "test": "jest",
+    "test:coverage": "jest --coverage",
+    "prepublish": "yarn run compile",
+    "lint": "eslint src",
```
Comment (sloops77) [inline packages/stellar-dispatcher/package.json]: tests

File: packages/stellar-dispatcher/package.json [hunk 8]
```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@stellarjs/stellar-dispatcher",
+  "version": "0.0.1",
+  "license": "MIT",
+  "main": "lib-es6/index.js",
```
Comment (sloops77) [inline packages/stellar-dispatcher/package.json]: does it need to also be browser?

File: packages/stellar-dispatcher/package.json [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+    "suiteName": "tests",
+    "output": "../../.artifacts/test_results/redux-stellar.xml",
+    "usePathForSuiteName": "true"
+  },
+  "scripts": {
+    "ci": "yarn run lint:ci && yarn run test-coverage",
+    "clean": "rimraf lib lib-es6",
+    "compile": "yarn run compile:es6 && yarn run compile:umd",
+    "compile:es6": "babel ./src  --out-dir ./lib-es6",
+    "compile:umd": "webpack -p",
```
Comment (sloops77) [inline packages/stellar-dispatcher/package.json]: if you are compiled umd, then you need a webpack.config.js

File: packages/stellar-dispatcher/package.json [hunk 10]
```diff
@@ -0,0 +1,50 @@
+{
```
Comment (sloops77) [inline packages/stellar-dispatcher/package.json]: also you need an npmignore

File: packages/stellar-dispatcher/package.json (hunk 8 above)
Comment (roeeyud) [reply packages/stellar-dispatcher/package.json]: nope

File: packages/stellar-dispatcher/src/index.js (hunk 1 above)
Comment (roeeyud) [reply packages/stellar-dispatcher/src/index.js]: Done

File: packages/stellar-dispatcher/src/index.js (hunk 6 above)
Comment (roeeyud) [reply packages/stellar-dispatcher/src/index.js]: What?

File: packages/stellar-dispatcher/src/index.js (hunk 2 above)
Comment (roeeyud) [reply packages/stellar-dispatcher/src/index.js]: I have no idea

File: packages/stellar-dispatcher/src/index.js (hunk 4 above)
Comment (roeeyud) [reply packages/stellar-dispatcher/src/index.js]: you mean path ? `${resource}:${path}` : resource?

File: packages/stellar-dispatcher/package.json (hunk 10 above)
Comment (roeeyud) [reply packages/stellar-dispatcher/package.json]: I have no idea what you are writing

**Pull request closed without merging**